exercises/branching-conditionals/rockPaperScissorsWinner/rockPaperScissorsWinner.js

```diff
@@ -24,15 +24,51 @@
  * @returns {string} Returns 'win', 'lose', or 'draw' depending on
  *  whether the first player won, lost, or drew the match.
  */
+let readlinesync = require('readline-sync')
+
+firstPlayer = readlinesync.question ('What is your name, Player One?') //user input
+console.log (firstPlayer + ' is Player One.')
+
+secondPlayer = readlinesync.question ('What is your name, Player Two?') //user input
+console.log (secondPlayer + ' is Player Two.')
+
+firstPlayerMove = readlinesync.question ('What is your move Player One? Enter 0 for scissors, 1 for paper, or 2 for rock.')
+//now i need to push these values into an array
+
+
 function rockPaperScissorsWinner(array) {
-  // This is your job. :)
-}
+  let firstPlayerMove = array[0]
+  let secondPlayerMove = array[1]
+  //something in here for the readlinesync? 
+  let result = '';
+  console.log('The array is:' + array)  
+  console.log('We are about to enter the loop')
+   console.log ('We are inside the for loop') 
+    if ((array === 0,1) || (array === 1,2) || (array === 2,0)) {
```

Comment:
The syntax `array === 0,1` isn't valid, I don't think. What I think you meant to write was:

```javascript
array === [0,1]
```

That doesn't work in JS, unfortunately. You can't compare two arrays directly like that (it'll always return `false`).

The only way around is to compare the elements one at a time. You could write your own function `arrayEquals` that does what you expect and use it here, though.

Reply:
I think @rachel-mclean wrote an `arrayEquals` function at my urging.